doc/mcuboot/conf.py

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
 #!/usr/bin/env python3
```

Comment:
I wonder if these scripts will work if the Shebang line `#!/usr/bin/env python3` isn't the first line of the file and python script has a C style comments ?